### ISSUE TITLE
Add config for periodic donation banner

### ISSUE THREAD

**Issue by virgile-dev** [umap-project/umap#1850]
**Is your feature request related to a problem? Please describe.**
Most people don't know they can contribute by donating money to uMap maintainers. The info is only available on umap-project.org and Github.

**Describe the solution you'd like**
Thunderbird's periodic full screen donation banner has shown outstanding results. See their [FOSDEM talk](https://fosdem.org/2024/schedule/event/fosdem-2024-2741-take-your-foss-project-from-surviving-to-thriving/) about it. They also share their choices in term of donation stack. 
I'd like for uMap to try and implement a similar banner. 
It could be a configuration option that can be enabled or disabled by the instance admin. 
I guess it would be interesting to be able to define the periodicity by which it would be displayed to users.
Of course we'd have to determine where we'd display it, I'm guessing not on iframes.
Also we'd need great design (ping @Aurelie-Jallut) and copy (ping @C-Sophie)  writing and translation in many languages.

Looking forward to see the discussion in the comment section to see if we can figure this out.

**Comment by Aurelie-Jallut** [comment]
Do you have a screenshot of the thunderbird banner?

**Comment by virgile-dev** [comment]
Yes there you go : https://www.thunderbird.net/fr/thunderbird/115.0/holidayeoy/

**Comment by yohanboniface** [comment]
A few more ideas from a discussion with @davidbgk:

- we want to be less intrusive as possible, but still find a way to be educative (open source has costs)
- maybe we could display a message on the edit panel when someone creates a new layer while having already (say) 10 layers: "Hey, you are enjoying uMap what about a small donation ?" and a few links to donate to developers, hosters…
- same idea, but when creating a new map,while having already xx maps
- or maybe after a save (maybe first save of first map, and first save of very 10 maps or something)
- those links should be configurable per uMap instance, so Framasoft can point to their association, etc.  And it should be deactivable
